ui/app/AppLayouts/Profile/views/profile/MyProfileSettingsView.qml

```diff
@@ -249,7 +248,7 @@ ColumnLayout {
             id: profileShowcaseAssetsPanel
             Layout.minimumHeight: implicitHeight
             Layout.maximumHeight: implicitHeight
-            baseModel: root.walletAssetsStore.groupedAccountAssetsModel
+            baseModel: root.walletAssetsStore.groupedAccountAssetsModel // TODO: separated model instance from profile section
```

Comment:
Could you elaborate more on the TODO please, I think `// TODO: separated model instance from profile section` seems very cryptic to me... 😄

Reply:
fixed on `instantiate an assets model in profile module`. it's in my next PR :)